qmi/instruments/newport/tlb670x.py

```diff
@@ -111,16 +111,15 @@ def _send(self, command_string: str) -> str:
         response = self._receive()
         if "?" in command_string:
             # Check if response string consists of multiple responses, separated by "\r\n".
-            if len(response) > 2 and len(response[-2]) > 0 and response[-2] != "OK":
```

Comment:
what prompted this change?

Reply:
There was an issue with this driver on a setup. We tested a bit more extensively which responses gave the issues and noticed that there are sometimes empty string in between. So, it was a rather "easy" fix so it got sneaked into this ticket so that there's no need to do another ticket+PR.